web/src/components/storage/SpaceActionsTable.tsx

```diff
@@ -43,8 +43,25 @@ import {
 } from "~/components/storage/device-utils";
 import { Icon } from "~/components/layout";
 import { PartitionSlot, SpacePolicyAction, StorageDevice } from "~/types/storage";
+import { configModel } from "~/api/storage/types";
 import { TreeTableColumn } from "~/components/core/TreeTable";
 import { Table, Td, Th, Tr, Thead, Tbody } from "@patternfly/react-table";
+import { useConfigModel } from "~/queries/storage/config-model";
+
+const isUsedPartition = (partition: configModel.Partition): boolean => {
```

Comment:
NP: these are the things I still envision in a storage utils module. But it's hard to convince you guys.

Reply:
Is not that we don't agree. There is also a comment right below that like saying tht we need to reconsider all that. It currently feels like dealing with the old YaST huge storage "maps" (pre storage-ng).